Tests/UnitTests/Plugins/GeoModel/GeoDetectorObjectTest.cpp

```diff
@@ -69,16 +71,19 @@ void test(const Acts::GeoModelDetectorObjectFactory::Cache& cache,
         BOOST_CHECK(geoDims.boxI[0] == csxmax);
         BOOST_CHECK(geoDims.boxI[1] == csymax);
       }
+
+      if (sbounds.type() != Acts::SurfaceBounds::eTrapezoid) {
+        continue;
+      }
+
       // trap Surface without trf
-      if (sbounds.type() == Acts::SurfaceBounds::eTrapezoid) {
-        const auto* trapBounds =
-            dynamic_cast<const Acts::TrapezoidBounds*>(&sbounds);
-        std::vector<Acts::Vector2> trapVerts = trapBounds->vertices();
-
-        for (long unsigned int i = 0; i < trapVerts.size(); i++) {
-          BOOST_CHECK(trapVerts[i][0] == geoDims.trapVerts[i][0]);
-          BOOST_CHECK(trapVerts[i][1] == geoDims.trapVerts[i][1]);
-        }
+      const auto* trapBounds =
+          dynamic_cast<const Acts::TrapezoidBounds*>(&sbounds);
+      std::vector<Acts::Vector2> trapVerts = trapBounds->vertices();
+
+      for (long unsigned int i = 0; i < trapVerts.size(); i++) {
```

Comment:
`long unsigned int` is an interesting type 😄

Reply:
Wenn man `std::size_t` bei Wish.com bestellt.

Reply:
Can you wait 10 weeks to receive your order?
https://github.com/acts-project/acts/pull/3930